kokoro-tts

```diff
@@ -923,7 +923,7 @@ if __name__ == "__main__":
         sys.exit(1)
 
     # Ensure the input file exists
-    if not os.path.isfile(input_file):
+    if not os.access(input_file, os.R_OK):
```

Comment:
good catch!

Reply:
The error message "The file does not exist" might be misleading in some cases where the file exists but isn't readable.

i'll update it